shared/js/media/video_player.js

```diff
@@ -471,6 +483,20 @@ function VideoPlayer(container) {
     }
   });
 
+  slider.addEventListener('keypress', function(e) {
+    // Currently, we only allow screen reader users to adjust sliders with a
+    // constant step size (there is no page up/down equivalent). In the case
+    // of videos, we make sure that the maximum amount of steps for the entire
+    // duration is 20, or 2 second increments if the duration is less then 40
+    // seconds.
```

Comment:
Please expand this comment a bit to actually say that even though we're on a device without arrow keys, screen readers on the device will send up and down key events to control sliders like this.  Otherwise future maintainers of the app won't understand this.